wp-content/themes/core/pcss/theme/components/_all.pcss

```diff
@@ -5,4 +5,3 @@
 @import "content-block";
```

Comment:
Let's go ahead and nuke the `/pcss/theme/` directory altogether at this point. We'll work through v2.0 as part of this initiative (if you've already started doing work on the buttons, feel free to hoist that work and move it down directly into the core pcss) ; )

Reply:
Moved the partials up and nixed the theme directory